src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -120,12 +120,12 @@ public class C
           ""locations"": [
           ],
           ""fullMessage"": ""Program does not contain a static 'Main' method suitable for an entry point"",
+          ""isSuppressedInSource"": false,
           ""properties"": {{
             ""severity"": ""Error"",
             ""defaultSeverity"": ""Error"",
             ""category"": ""Compiler"",
             ""isEnabledByDefault"": ""True"",
```

Comment:
The spec says "Every JSON _property name_ [emphasis added] in the SARIF format shall be a camelCase name". There are two issues:

1. The spec implies, but doesn't state explicitly, that this applies to property bag properties as well (for example, `isEnabledByDefault", `defaultSeverity"). I'll make that explicit.

2. The spec doesn't say anything at all about the _values_ of the properties, which is what we're concerned about here, right? You're asking whether the value for `isEnabledByDefault` should have been `true`, and whether the value of `customTags` should have been `compiler;telemetry;notConfigurable`, right?

It that's the question, my answer is that I really don't care. Property values might be anything: strings, numbers, code fragments in some arbitrary language like JavaScript, _etc_. I'd rather not get into the business of legislating what they look like.